### ISSUE TITLE
Don't send email address to Stripe

### ISSUE THREAD

**Issue by adriaanvanrossum** [simpleanalytics/roadmap#17]
We send emails to Stripe to find customer details quick, but we don't have to. Let's change that.

It also means that we need to change other stuff

- [ ] Send payment reminders ourselves
- [ ] Sync a `stripe_uuid` or something with Stripe so we can still find the user

**Comment by LeCodeCo** [comment]
I believe you can still get webhooks when Stripe would send a payment reminder, so that'd be pretty simple.

**Comment by adriaanvanrossum** [comment]
This is true, we indeed need to use webhooks. We already use them to forward the invoices to customize the emails customers get.
